src/include/index/generic_key.h

```diff
@@ -67,11 +67,10 @@ class GenericKey {
    * @return
    */
   const std::string GetInfo() const {
-    storage::Tuple tuple(schema, data);
+    storage::Tuple tuple(schema, (char *)data);

```

Comment:
This is the bug in Peleton logging, otherwise we don't want to change this file.